src/pages/workflow-execution/WorkflowExecution.tsx

```diff
@@ -187,16 +187,12 @@ export const WorkflowExecution = () => {
         sortable
         field="currentStep"
         header="Step"
-        style={{ width: '15%' }}
+        style={{ width: '13%', maxWidth: '13%' }}
         body={({ currentStep }) => (
           <>
             <Tooltip target=".current-workflow-step" />
             <span
               className="current-workflow-step column-cut-text"
-              style={{
-                minWidth: '75px',
-                maxWidth: '200px',
-              }}
```

Comment:
2 different widths (1 on column, 2 in cell) were destroying alignment with columns headers